com.github.btpf.alexandria.json

```diff
@@ -0,0 +1,51 @@
+{
+  "id": "com.github.btpf.alexandria",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "42",
+  "sdk": "org.gnome.Sdk",
+  "command": "alexandria",
+  "rename-icon": "alexandria",
+  "separate-locales": false,
+  "finish-args": [
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--share=ipc",
+    "--share=network",
+    "--device=dri",
+    "--filesystem=/run/media"
+  ],
+  "modules": [
+    {
+      "type": "file",
+      "name": "alexandria.deb",
+      "buildsystem": "simple",
+      "build-commands": [
+        "ar x ./alexandria.deb",
+        "tar -xf ./data.tar.gz",
+        "ls",
+        "install -Dm755 -t /app/bin/ usr/bin/alexandria",
+        "mkdir -p /app/share/icons/hicolor",
+        "cp -r usr/share/icons/hicolor/* /app/share/icons/hicolor/",
+        "mv /app/share/icons/hicolor/256x256@2/ /app/share/icons/hicolor/256x256/",
+        "rm control.tar.gz",
+        "rm data.tar.gz",
+        "rm debian-binary",
+        "rm -rdf usr",
+        "rm alexandria.deb",
+        "install -Dm644 -t /app/share/applications/ com.github.btpf.alexandria.desktop"
+      ],
+      "sources": [
+        {
+          "type": "file",
+          "url": "https://github.com/btpf/Alexandria/releases/download/0.10.0/alexandria_0.10.0_amd64.deb",
```

Comment:
Looks like this is x86-64 only? Are you able to provide aarch64 binaries or build from source?

Reply:
If not: https://docs.flathub.org/docs/for-app-authors/maintanance#limiting-the-set-of-architectures-to-build-on

Reply:
build from source is always preferable.

Reply:
this is another blocker, because you build an aarch64 flatpak with a x86_864 binary. That's not gonna work. You should build from source here since Github still doen't support aarch64.